src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { NavLink, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeopleTable = () => {
+  const { personSlug } = useParams();
+
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
you can set loading to true initially, and when you're done fetching data you can set it to false